.changeset/nice-snakes-camp.md

```diff
@@ -0,0 +1,90 @@
+---
+"@salt-ds/core": minor
+---
+
+**_Theming and CSS updates_** with visual changes to Salt components:
+
+1. Minor changes in `Button` component width. `CTA` and `Primary` button text font weight changed from bold to semibold. Expect components containing Salt `Button` to have similar change.
+
+```diff
+- --button-fontWeight: var(--salt-actionable-cta-fontWeight);
+- --button-fontWeight: var(--salt-actionable-primary-fontWeight);
+- --button-fontWeight: var(--salt-actionable-secondary-fontWeight);
++ --button-fontWeight: var(--salt-text-action-fontWeight);
+```
+
+<!-- INSERT IMAGE SHOWCASING DIFFERENCES HERE -->
+
+2. Disabled `InteractableCard` component has border color updated from blue to grey.
+
+```diff
+- border-color: var(--saltCard-borderColor-disabled, var(--salt-accent-borderColor-disabled));
++ border-color: var(--saltCard-borderColor-disabled, var(--salt-container-primary-borderColor-disabled));
```

Comment:
again not sure if this diff makes much of a difference. the change is more the switch from an accent token to container disabled token which needs to be expressed in the above change